.github/workflows/build-website.yml

```diff
@@ -40,6 +40,29 @@ jobs:
       - name: bundle site
         run: cd website && zip -r ../website.zip www
 
+      - name: set up check-links
+        run: |
+          python3.7 -m venv ENV
+          ENV/bin/pip install bs4 requests
+      - name: check links (dev)
+        run: |
+          cd website/
+          yarn start localhost 9001 &
```

Comment:
why start the server inside website/ ? it should be website/public
something is wrong, its not reporting errors that way

Reply:
When I develop the website, I always run `yarn start` from within website/.

The website/public/ directory contains the page sources, but doesn't contain the server entry point. The server entry point is website/run-server.mjs.

Reply:
ok got. so yarn sets the entry point. i was using `python3 -m http.server 8080` inside /website/

Reply:
Ah, I see why you were confused. The website's guts have been restructured since you made check-links.